src/index.html

```diff
@@ -11,8 +11,51 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
Stylesheets with .scss extension can't be included directly in HTML. They need to be compiled into .css first. You should link to a .css file here.